packages/twenty-chrome-extension/src/options/Options.tsx

```diff
@@ -4,24 +4,31 @@ import styled from '@emotion/styled';
 import { Loader } from '@/ui/display/loader/components/Loader';
```

Comment:
Could we rename Options into a more self-explanatory naming ? Options seems very vague, SidePanel or something like that would be more appropriate.

Reply:
Done 👍